src/store/getters.ts

```diff
@@ -699,19 +697,20 @@ export const getEntityTypeOptions = (state: StateIF): Array<EntityI> => {
   })
 
   let options: SelectOptionsI[]
-  // special case for amalgamation - location may be CA or IN
-  if (isAmalgamation) options = [...bcOptions]
+  // special case for amalgamation - ignore location
+  if (isAmalgamation(state)) options = [...bcOptions]
```

Comment:
Ignore location because it hasn't been set yet (for xpro aml, location comes after entity type).